startup.sh

```diff
@@ -48,8 +43,17 @@ DEFAULT_HEALTHCHECK_URL="http://${DEFAULT_HEALTHCHECK_URL}"
 # write the healthcheck URL to a file that geoserver user has access to but is not served by tomcat
 echo "${HEALTHCHECK_URL:-$DEFAULT_HEALTHCHECK_URL}" > $CATALINA_HOME/conf/healthcheck_url.txt
 
+# set geoserver require file to global.xml
+if [ "SET_GEOSERVER_REQUIRE_FILE" = true ]; then
```

Comment:
Please fix this to `$SET_GEOSERVER_REQUIRE_FILE`
Currently no ENV is used here :-)

Reply:
oups :D